src/main/java/com/infernalsuite/benchmark/commands/sub/LoadCommand.java

```diff
@@ -0,0 +1,49 @@
+package com.infernalsuite.benchmark.commands.sub;
+
+import java.io.IOException;
+
+import com.infernalsuite.asp.api.AdvancedSlimePaperAPI;
+import com.infernalsuite.asp.api.exceptions.CorruptedWorldException;
+import com.infernalsuite.asp.api.exceptions.NewerFormatException;
+import com.infernalsuite.asp.api.exceptions.UnknownWorldException;
+import com.infernalsuite.asp.api.loaders.SlimeLoader;
+import com.infernalsuite.asp.api.world.SlimeWorld;
+import com.infernalsuite.asp.api.world.SlimeWorldInstance;
+import com.infernalsuite.asp.api.world.properties.SlimePropertyMap;
+import com.infernalsuite.benchmark.commands.ASPBenchmarkCommand;
+import com.infernalsuite.benchmark.commands.exception.MessageCommandException;
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.format.NamedTextColor;
+import org.bukkit.command.CommandSender;
+import org.incendo.cloud.annotations.Argument;
+import org.incendo.cloud.annotations.Command;
+
+public class LoadCommand {
+
+    private final ASPBenchmarkCommand manager;
+
+    public LoadCommand(ASPBenchmarkCommand manager) {
+        this.manager = manager;
+    }
+
+    @Command("aspbenchmark|aspb|swmb load <world>")
```

Comment:
Maybe we can also have a command that does iterations here? Just a suggestion tho 

Basically like this:
- read world
- for n iterations:
  - start measurement
  - loadWorld
  - pause measurement
  - unload World
  - reiterate